### ISSUE TITLE
Over-time smoothing

### ISSUE THREAD

**Issue by sammo3182** [fsolt/dcpo_dem_mood#20]
R2: I am concerned about the over-time smoothing induced by the random-walk prior in measurement model. In county-years without survey data, this prior will impute the latent variable by interpolating between adjacent years, and in years with data it may still make over-time changes smoother than the raw data suggest. From a descriptive perspective this is largely a virtue, but in models that rely for causal identification on sharp over-time shifts, such as the thermostatic model, it can be a big problem. There is no magic fix for this problem, but readers should still be given more information on how much "work" the prior is doing—what proportion of country-years are missing data, how informative is the prior relative to the likelihood, and so on.

**Comment by fsolt** [comment]
Yeah, we're going to have to talk about this.  We want to make clear to readers that this is a characteristic of Claassen's data, not anything we've done (see [this point](https://github.com/fsolt/dcpo_dem_mood/issues/18#issuecomment-8244651220)), and that it could be influencing the results.

One possibility that occurs to me as a way of getting a handle on the influence of the interpolated country-years is to just drop them from the analysis dataset.  Of course then we'd have to be sure our method won't mind the breaks, but still.

**Comment by casstai** [comment]
I added some discussion on the random walk of priors. However,  now I am thinking maybe we don't need this discussion because R2's concern is about the domination of priors in the case of sparse data.  More explicitly,  R2 is considering the effects of different priors. But, in our revised piece,  we only used Model5, and expanded data should reduce the concern about prior's domination.  So, our results just enforced our point about uncertainty. Random walk of prior can explain the null result in M2 and M3 but cannot explain the significance in M1. 

I guess my question is, do we still need to discuss it and show missing country-years, like using this point to support our null result?
